package.json

```diff
@@ -25,6 +25,7 @@
     "clean-tag": "2.0.2",
     "cross-fetch": "2.2.3",
     "dotenv": "6.1.0",
+    "draft-js": "^0.10.5",
```

Comment:
For now, we use fixed versions for main dependencies.

Reply:
Fixed